tests/PCA.py

```diff
@@ -0,0 +1,60 @@
+import pandas as pd
+import numpy as np
+from sklearn.decomposition import PCA
+from sklearn.preprocessing import StandardScaler
+import matplotlib
+import matplotlib.pyplot as plt
+
+
+# Reading the data
+
+num_components = 2
+#importing 1112X9 data frame
+path1 = "1112x9.csv"
+#importing 17X9 data frame
+path2 = "17x9.csv"
+df1 = pd.read_csv(path1)
+df2 = pd.read_csv(path2)
+
```

Comment:
Please move all this to a separate PCA_test.py file, where the PCA function is run on actual data. You can check how test files for other functions look like for reference. Please also provide either the csv-file or hard code the data directly into the dataframe.